apiary.apib

```diff
@@ -66,81 +99,136 @@ password as `x-password` header.
 
 ### Sign Up [PUT]
```

Comment:
Actually, I read into POST vs PUT a bit more and decided that actually PUT makes sense for a couple of the calls.  JSON-API doesn’t forbid the use of PUT, it just promotes the use of PATCH for patch updates and forbids using PUT for that purpose (which should actually send a full payload).